Allure.NUnit/Core/AllureExtensions.cs

```diff
@@ -113,6 +114,76 @@ public static T WrapInStep<T>(this AllureLifecycle lifecycle, Func<T> func, stri
             }
         }
 
+        /// <summary>
+        /// Wraps async Action into AllureStep.
+        /// </summary>
+        [Obsolete("Use [AllureStep] method attribute")]
```

Comment:
There is no need to add the already obsolete method.

Reply:
I've removed [Obsolete] - Maybe you will set [Obsolete] on them later in next major release? :)